assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -173,4 +173,19 @@ export const isMobile = {
 // works for SCREAMING_SNAKE_CASE, snake_case or cRazY_SNAKE_case
 export const snakeToCamel = (string: string) => string.toLowerCase().replace(/_[a-z]/g, match => match[1].toUpperCase());
 
-export const youtubeRegexp = /(?:youtube\.com\/(?:[^/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?/ ]{11})/i;
\ No newline at end of file
+export const youtubeRegexp = /(?:youtube\.com\/(?:[^/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?/ ]{11})/i;
+
+export const getCookieItem = (sKey: string) => {
+  if (!sKey) {
+    return null;
+  }
+  return (
+    decodeURIComponent(
+      document.cookie.replace(
+        // eslint-disable-next-line no-useless-escape
```

Comment:
are there useful escapes that eslint consider as useless ?!

Reply:
I did not take the time to try removing them and test if the function still works in all main cases... should I?

Reply:
You should. You normally don't have to espace with '\' all characters in brackets.